tests/test_oml/test_datasets/test_datasets.py

```diff
@@ -13,6 +13,10 @@
     TEXTS_COLUMN,
```

Comment:
Is that possible to make one more test file under `.../test_datasets/test_audio.py` and check audio specific stuff?
We can try to instantiate dataset items with different values of `sr`, `num_seconds`, offsets and perform other checks that make sense to you.

Reply:
done